homeassistant/components/emulated_hue/__init__.py

```diff
@@ -111,14 +111,14 @@ async def async_setup(hass, yaml_config):
 
     DescriptionXmlView(config).register(app, app.router)
     HueUsernameView().register(app, app.router)
+    HueConfigView(config).register(app, app.router)
```

Comment:
Does this need to be re-ordered?

Reply:
Yes, config needs to be registered before "unauthorized user" or it will reject the cofig